ckanext/switzerland/templates/snippets/add_source_button.html

```diff
@@ -0,0 +1,9 @@
+{% set authorized_user = h.check_access('sysadmin') %}
```

Comment:
it might be nice to check for permission in the same way in each occurrence. but then, we will probably only stumble over this, when we compare them in this PR-view 😄 
if you feel like changing it to be consistent, do so, but if you feel there is no real value, feel free to merge it 🙂